web/src/api/images/get_uploaded_image.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  errorHasuraQuery,
+  errorNotAllowed,
+  errorResponse,
+} from "src/backend/errors";
+import { NextApiRequest, NextApiResponse } from "next";
+import * as yup from "yup";
+import { protectInternalEndpoint } from "src/backend/utils";
+import { getAPIServiceGraphqlClient } from "src/backend/graphql";
+import { getSdk as checkUserInAppDocumentSDK } from "@/api/images/graphql/checkUserInApp.generated";
+import { S3Client, GetObjectCommand } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { logger } from "@/lib/logger";
+
+type RequestQueryParams = {
+  app_id: string;
+  image_type: string;
+  content_type_ending: string;
```

Comment:
wouldn't we want to get this automatically from Hasura? seems like it would not be great UX otherwise